courses/free-courses-id.md

```diff
@@ -98,6 +98,7 @@
 * [Flutter Membangun Website](https://buildwithangga.com/kelas/flutter-membangun-website-sederhana?thumbnail=nk4neM2UyG.110&main_leads=browse) - BuildWithAngga
 * [Flutter Tutorial (Flutter Fundamentals)](https://www.youtube.com/playlist?list=PLZQbl9Jhl-VACm40h5t6QMDB92WlopQmV) - Erico Darmawan Handoyo
 * [Tutorial Flutter](https://www.youtube.com/playlist?list=PL0-7Xi0GB3teRqkuBusUEcVrP6OlYpD9w) - idr corner
+* [STUDI KASUS FLUTTER + GETX + FIREBASE [ CHAT APPS 2021 ]](https://www.youtube.com/playlist?list=PL7jdfftn7HKt6wPnVXoXgserU14d_ACA-) - Sandikha Rahardi, Kuldii Project (YouTube)
```

Comment:
And as I suggest you in previous review... reorder to satisfy linter list alphabetization.

Check CI logs (Show all checks > Details):
https://github.com/EbookFoundation/free-programming-books/pull/5601/checks?check_run_id=3762737097
```logs
Run fpb-lint ./courses/
/courses/free-courses-id.md
  98:1-101:180  warning  Alphabetical ordering: swap l.101 and l.100  alphabetize-lists  remark-lint
```